site/source/docs/porting/exceptions.rst

```diff
@@ -149,7 +150,8 @@ this option.
 
 If the stack pointer has been moved due to stack allocations within the Wasm
 function before an exception is thrown, you can to use ``stackSave()`` and
-``stackRestore()`` to restore the stack pointer so no stack memory is leaked.
+``stackRestore()`` to restore the stack pointer so that no stack memory is
+leaked.
```

Comment:
It looks like the current published website doesn't include the `stackSave`/`stackRestore` at all: https://emscripten.org/docs/porting/exceptions.html

Maybe its just out of date?  @kripken could you update after this change lands?